### ISSUE TITLE
Display interactive Vega-Lite charts in HTML output -- instead of static PNG

### ISSUE THREAD

**Issue by Juan-132** [jupyter/nbconvert#1198]
I hope this is the right place to continue this discusion. https://github.com/altair-viz/altair/issues/329

Currently, NBConvert includes static PNG images in the HTML output, if the notebook includes interactive Vega-Lite charts. This is unfortunate, as interactive Vega Lite charts have a lot of added value.

As of Altair 4.0, the default renderer for Vega-Lite is HTML-based. This allows for creating HTML output with interactive Vega-Lite charts.

Questions:

- How can this functionality be included in NBConvert?
- Is there any workaround available that we could use right now?

Example code using the [Altair Chart Save API](https://altair-viz.github.io/user_guide/generated/toplevel/altair.Chart.html#altair.Chart.save).

```python
import pandas as pd
import altair as alt
from vega_datasets import data
cars = data.cars()

cars = alt.Chart(cars).mark_point().encode(
    x='Horsepower',
    y='Miles_per_Gallon',
    color='Origin',
    tooltip=['Name', 'Origin']
).interactive()

cars.save('cars.html')
```

**Comment by MSeal** [comment]
> Currently, NBConvert includes static PNG images in the HTML output, if the notebook includes interactive Vega-Lite charts. This is unfortunate, as interactive Vega Lite charts have a lot of added value.

Agreed the html version is better! When I run your example above with

```
cars
```

in a following cell for both nbconvert 5.6.1 and 6.0.0a0 I actually do get a nice HTML output from `jupyter nbconvert cars.ipynb --to=html`. Do you know the constraints / environment under which it's only rendering a PNG?

**Comment by Juan-132** [comment]
Hi @t-makaro 
Thanks for taking the effort to investigate
I tried to reproduce the issue, and it's gone. Interactive charts work great in NBConvert. Which is splendid, and makes NBConvert an even greater package! Thanks for all the effort you guys put in.

I believe the issue was caused by me having an older version of Altair installed (which I was not aware of untill recently). Conda/main is still on v3.2, while conda-forge does have the latest package. https://github.com/uwdata/visualization-curriculum/issues/11
